mycpp/gc_mylib.cc

```diff
@@ -1,16 +1,31 @@
 // gc_mylib.cc
 
-#include "gc_mylib.h"
+#include "mycpp/runtime.h"
 
 #include <errno.h>
 #include <unistd.h>  // isatty
 
-#include "gc_builtins.h"
-
 mylib::BufWriter gBuf;
 
+
+// NOTE(Jesse): This was literally the only thing left in gc_builtins.cc so I
```

Comment:
I think we should still have some sort of `builtins.{cc,h} `   file

i.e. the structure of containers/builtins/mylib still makes sense